minorminer/utils/subgraph.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2023 D-Wave
```

Comment:
We can delete this module, looks like you successfully removed the dependency in raster_embedding